risc0/binfmt/Cargo.toml

```diff
@@ -8,8 +8,9 @@ homepage = { workspace = true }
 repository = { workspace = true }
 
 [dev-dependencies]
+env_logger = "0.10"
```

Comment:
why is both env_logger and tracing subscriber a part of this? I thought that we only needed `tracing` and `tracing-subscriber`